src/data/types/duration.ts

```diff
@@ -0,0 +1,164 @@
+const millisecond = 1;
+const microsecond = millisecond / 1000;
+const nanosecond = microsecond / 1000;
+const second = 1000 * millisecond;
+const minute = 60 * second;
+const hour = 60 * minute;
+const day = 24 * hour;
+const week = 7 * day;
+
+const units = new Map([
+	["ns", nanosecond],
+	["µs", microsecond],
+	["μs", microsecond],
```

Comment:
Duplicated

Reply:
Try it!
```js
"µs" == "μs"
```
<img width="102" alt="image" src="https://github.com/surrealdb/surrealdb.js/assets/17960250/f342a2be-ecbb-4723-8555-a3e5ce2371dd">

Reply:
So I would add a comment